operator/document/v0/execution/pdf_checker.py

```diff
@@ -0,0 +1,21 @@
+from io import BytesIO
+import json
+import base64
+import sys
+
+# TODO: Deal with the import error when running the code in the docker container
```

Comment:
@chuang8511 
I think this PR is not ready, right?

Reply:
> # TODO: Deal with the import error when running the code in the docker container

I will deal with this part with other PR.
But, this PR is not ready because there is a bug in the container for prod. I may need to modify this PR. So, I draft it first.